packages/@ama-sdk/core/src/plugins/simple-api-key-authentication/readme.md

```diff
@@ -0,0 +1,58 @@
+## Simple API Key Authentication
+
+Plugin to handle the Simple API key authentication with Apigee.
+Note that this authentication mode has to be specifically activated per application and will be rejected by default.
+
+Provide the clientId of your application as the `apiKey`.
+It can be a simple *string* or a function that returns a *string* or a *Promise* if your application
+needs to change it at runtime.
+You can also call the method `setApiKey` to change it programmatically.
+
+The plugin `options` allow to customize the headers in which the plugin injects the API key and context overrides.
+
+About context overrides: today only the Office ID is supported.
+Like the `apiKey`, `options.officeId` can be a *string* or a function that returns a *string* or a *Promise*.
+You can also call the method `setOfficeId` to change it programmatically.
```

Comment:
maybe a word to differentiate both use cases (one where the user impacts the initialization and the other dynamic) ?